Sources/Data/OAAppData.h

```diff
@@ -53,6 +53,11 @@
 @property (nonatomic) BOOL mapillary;
 @property (readonly) OAObservable* mapillaryChangeObservable;
 
+@property (nonatomic) BOOL wikipedia;
+@property (readonly) OAObservable* wikipediaChangeObservable;
+@property (readonly) OACommonBoolean *wikipediaGlobalProfile;
```

Comment:
Preferences should be private (see how all other preferences implemented in OAAppData)